packages/record/src/context.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+
+export const RecordFieldContext = React.createContext();
```

Comment:
let's change this name since it coupling it to the RecordField, we can call it simply Context